### PR TITLE
Add countries for West Africa from issue #8

### LINKED ISSUE #8: Add West Africa Countries
If you're working on this ticket leave a comment, so no two people do the same work. After 2h of the comment, I'll start accepting other volunteers.

The PR to fix this issue should include all the countries with all the data.

# List of countries
Benin
Burkina Faso
Cabo Verde (Cape Verde)
Côte d'Ivoire (Ivory Coast)
The Gambia
Ghana
Guinea
Guinea-Bissau
Liberia
Mali
Mauritania
Niger
Nigeria
Saint Helena, Ascension and Tristan da Cunha (United Kingdom Overseas Territory)
Senegal
Sierra Leone
Togo

### PR DESCRIPTION
**Pull request by shakirandagire**
Closes [#8](https://github.com/lsmoura/countries/issues/8)

### CODE REVIEW

File: data/civ.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: Côte d'Ivoire
+official_name: 	The Republic of Côte d'Ivoire
```
Comment (lsmoura) [inline data/civ.yaml]: Can you remove the extra spaces, please?
```suggestion
official_name: The Republic of Côte d'Ivoire
```

File: data/shn.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+name: Saint Helena 
+Ascension Island Ascension Island
+Tristan da Cunha Tristan da Cunha
```
Comment (lsmoura) [inline data/shn.yaml]: These lines should not be here

**Review by lsmoura — CHANGES_REQUESTED** [review]

File: data/civ.yaml (hunk 1 above)
Comment (shakirandagire) [reply data/civ.yaml]: Am on it

File: data/shn.yaml (hunk 2 above)
Comment (shakirandagire) [reply data/shn.yaml]: Let me remove them

**Comment by shakirandagire** [comment]
@lsmoura I have fixed the issues highlighted

**Review by lsmoura — APPROVED** [review]

**Pull request merged**